src/modules/claim/claim-verification.service.spec.ts

```diff
@@ -27,7 +27,7 @@ const resolvedCredential: RoleEIP191JWT = {
     'eyJhbGciOiJFUzI1NksiLCJ0eXAiOiJKV1QifQ.eyJjbGFpbURhdGEiOnsicmVxdWVzdG9yRmllbGRzIjpbXSwiY2xhaW1UeXBlIjoicHJlc2lkZW50LnJvbGVzLnN1Ym9yZ3Mud2hpdG5leS5pYW0uZXdjIiwiY2xhaW1UeXBlVmVyc2lvbiI6MSwiaXNzdWVyRmllbGRzIjpbXX0sImRpZCI6ImRpZDpldGhyOnZvbHRhOjB4MTdiNjVDOEM5NzQ2Rjg3YzgyY2M2ZjdDNEZDMzhmQ0E1ZjFBZUI3NSIsInNpZ25lciI6ImRpZDpldGhyOnZvbHRhOjB4MTdiNjVDOEM5NzQ2Rjg3YzgyY2M2ZjdDNEZDMzhmQ0E1ZjFBZUI3NSIsImNyZWRlbnRpYWxTdGF0dXMiOnsiaWQiOiJodHRwOi8vbG9jYWxob3N0OjMwMDAvdjEvc3RhdHVzLWxpc3QvdXJuOnV1aWQ6ZjhhZDAxZGYtZDA0YS00NjY3LTlmMzYtM2M3YTdlOTBhZDQ3IiwidHlwZSI6IlN0YXR1c0xpc3QyMDIxRW50cnkiLCJzdGF0dXNMaXN0Q3JlZGVudGlhbCI6Imh0dHA6Ly9sb2NhbGhvc3Q6MzAwMC92MS9zdGF0dXMtbGlzdC91cm46dXVpZDpmOGFkMDFkZi1kMDRhLTQ2NjctOWYzNi0zYzdhN2U5MGFkNDciLCJzdGF0dXNQdXJwb3NlIjoicmV2b2NhdGlvbiIsInN0YXR1c0xpc3RJbmRleCI6IjAifSwiZXhwIjo5MDA3MTk5MjU0NzQwOTkwLCJpc3MiOiJkaWQ6ZXRocjp2b2x0YToweDE3YjY1QzhDOTc0NkY4N2M4MmNjNmY3QzRGQzM4ZkNBNWYxQWVCNzUiLCJzdWIiOiJkaWQ6ZXRocjp2b2x0YToweDE3YjY1QzhDOTc0NkY4N2M4MmNjNmY3QzRGQzM4ZkNBNWYxQWVCNzUifQ.MHhiMzY2MjNjZDA3OTkwNjIwNTQwNTJkNjExZjFjMjQyM2UwNjgxNWJiNzRiMDU5ODc3ZWEwMTI3MjFiNTZhZDQ3MWEwNmFjYzc0Y2Y4OTFjZWEzNjdmMzRmZDQ5YzkxN2EwMGU2NWQyZGViMmNkNWY4N2M2MjE5ZWM5NjYyMjAyYjFi',
   payload: {
     claimData: {
-      fields: undefined,
+      requestorFields: undefined,
```

Comment:
Apparently updating of `passport-did-auth` caused updating of `@energyweb/vc-verification`, which allowed to detect this error